macros/edr/tests/test_json_schema.sql

```diff
@@ -0,0 +1,43 @@
+{% test test_json_schema(model, column, json_schema) %}
```

Comment:
I think the test should be just called `json_schema`, there's no need to re-specify that it's a test.